packages/api/test/nfts-store.spec.js

```diff
@@ -1,144 +1,158 @@
-        data: {
-          name: 'name',
-          description: 'stuff',
-          properties: {
-            extra: 'meta',
-            trick:
-              'ipfs://bafyreiemweb3jxougg7vaovg7wyiohwqszmgwry5xwitw3heepucg6vyd4',
-            src: [
-              'ipfs://bafybeifvbzj3rk2unsdhbq6wisbcblekwf2pjpgjmppv6ejplsyyhdn4ym/hello.txt',
-              'ipfs://bafybeibgaiw7jgzvbgjk3xu26scmbzedgywpkfgorrb7bfmu2hvpihzi5i/blob',
-            ],
-          },
-          image:
-            'ipfs://bafybeieb43wq6bqbfmyaawfmq6zuycdq4bo77zph33zxx26wvquth3qxau/cat.png',
+  t.deepEqual(
+    result,
+    {
+      ipnft: 'bafyreicnwbboevx6g6fykitf4nebz2kqgkqz35qvlnlcgfulhrris66m6i',
+      url: 'ipfs://bafyreicnwbboevx6g6fykitf4nebz2kqgkqz35qvlnlcgfulhrris66m6i/metadata.json',
+      data: {
+        name: 'name',
+        description: 'stuff',
+        properties: {
+          extra: 'meta',
+          trick:
+            'ipfs://bafyreiemweb3jxougg7vaovg7wyiohwqszmgwry5xwitw3heepucg6vyd4',
+          src: [
+            'ipfs://bafybeifvbzj3rk2unsdhbq6wisbcblekwf2pjpgjmppv6ejplsyyhdn4ym/hello.txt',
+            'ipfs://bafybeibgaiw7jgzvbgjk3xu26scmbzedgywpkfgorrb7bfmu2hvpihzi5i/blob',
+          ],
         },
+        image:
+          'ipfs://bafybeieb43wq6bqbfmyaawfmq6zuycdq4bo77zph33zxx26wvquth3qxau/cat.png',
       },
-      'response structure'
-    )
+    },
+    'response structure'
+  )
 
-    const { data, error } = await rawClient
-      .from('upload')
-      .select('*, content(cid, dag_size, pin(content_cid, status, service))')
-      .match({ content_cid: result.ipnft, user_id: userId })
-      .single()
+  const { data, error } = await getRawClient(config)
+    .from('upload')
+    .select('*, content(cid, dag_size, pin(content_cid, status, service))')
+    .match({ content_cid: result.ipnft, user_id: userId })
+    .single()
 
-    if (error) {
-      throw new Error(JSON.stringify(error))
-    }
+  if (error) {
+    throw new Error(JSON.stringify(error))
+  }
 
-    assert.strictEqual(data.type, 'Nft', 'nft type')
-    assert.strictEqual(data.content.dag_size, 324, 'nft size')
-    assert.deepStrictEqual(data.content.pin, [
-      {
-        content_cid:
-          'bafyreicnwbboevx6g6fykitf4nebz2kqgkqz35qvlnlcgfulhrris66m6i',
-        status: 'PinQueued',
-        service: 'IpfsCluster3',
-      },
-    ])
-  })
+  t.is(data.type, 'Nft', 'nft type')
+  t.is(data.content.dag_size, 324, 'nft size')
+  t.deepEqual(data.content.pin, [
+    {
+      content_cid:
+        'bafyreicnwbboevx6g6fykitf4nebz2kqgkqz35qvlnlcgfulhrris66m6i',
+      status: 'PinQueued',
+      service: 'IpfsCluster3',
+    },
+  ])
+})
 
-  it('should store dir wrapped image', async () => {
-    const { token, userId } = await createTestUser()
+// Miniflare doesn't currently preserve pathnames in File objects uploaded via FormData.
+// TODO: re-enable this test once https://github.com/cloudflare/miniflare/pull/309 is merged
```

Comment:
Can we open an issue in our repo pointing to that one?

Reply:
Good call - I made #2088 to remind us to re-enable once the bug is fixed.